scenarioo-client/app/shared/restServices.js

```diff
@@ -81,7 +81,7 @@ angular.module('scenarioo.services')
     })
 
     .factory('BuildReimportResource', function (ScenariooResource) {
-        return ScenariooResource('/builds/reimportBuild/:branchName/:buildName',
+        return ScenariooResource('/builds/:branchName/:buildName/import',
```

Comment:
we have to document the services that changed the endpoint under ur breaking changes